webbpsf/optics.py

```diff
@@ -773,6 +796,23 @@ def __init__(self, name="unnamed BLC", kind='nircamcircular', module='A', nd_squ
         else:
             self.bar_offset = None
 
+    def get_baroff_from_siaf(self, filt, channel='LW'):
```

Comment:
minor: `get_bar_offset_from_siaf` would be slightly more clear as a function name, since it's all real words (except for "siaf" of course)